Scheduled Repro Check Failed for Config

There was a failure of a monthly reproducibility check on ACCESS-NRI/access-om2-configs.
Logs, checksums and other artifacts can be found at the Failed Run Log link below.

Model: access-om2, found here: https://github.com/ACCESS-NRI/access-om2
Config Repo: access-om2-configs, found here: https://github.com/ACCESS-NRI/access-om2-configs
Config Tag Tested for Reproducibility: , found here: https://github.com/ACCESS-NRI/access-om2-configs/tags/
Failed Run Log: https://github.com/ACCESS-NRI/access-om2-configs/actions/runs/8104574328
Experiment Location (Gadi): /scratch/tm70/repro-ci/experiments/access-om2/release-1deg_jra55_ryf-1.0.0

Tagging @ACCESS-NRI/model-release

This is a weird error

https://github.com/ACCESS-NRI/access-om2-configs/actions/runs/8104574328/job/22151450280#step:3:133

```
>       with open(self.config_path) as f:
E       FileNotFoundError: [Errno 2] No such file or directory: '/scratch/tm70/***/test-model-repro/control/test_bit_repro_historical/config.yaml'

/g/data/tm70/repro-ci/test/access-om2-configs/test/exp_test_helper.py:28: FileNotFoundError
=========================== short test summary info ============================
FAILED ../../../../../../../g/data/tm70/repro-ci/test/access-om2-configs/test/test_bit_reproducibility.py::TestBitReproducibility::test_bit_repro_historical
======================= 1 failed, 3 deselected in 5.29s ========================
/bin/bash: line 31: --output-path: command not found
```

It didn't like the `--output-path` option?

--- a/config/released-configs.json
+++ b/config/released-configs.json
@@ -1,6 +1,6 @@
 {
     "$schema": "./released-configs.schema.json",
     "tags": [
-        "release-1deg_jra55_ryf-1.0.0"
+        "release-1deg_jra55_ryf-1.0"
     ]
 }

